### ISSUE TITLE
Fetus or baby

### ISSUE THREAD

**Issue by SusieWd** [nhsuk/nhsuk-service-manual-backlog#136]

**Comment by sarawilcox** [comment]
Discussed with Susie, Justine and Jean. Written up as follows: 

On the list of ages/life stages on our inclusive language page (see our existing page: https://beta.nhs.uk/service-manual/content/inclusive-language - Jean's working on an update), we should delete foetus but include:

embryo - up to 6 weeks
baby - during pregnancy, at birth and up to 1 year

(We agreed at the last Style Council meeting to use "toddler" for 1 to 3 years.)

I suggest we then link from "baby" here in the Inclusive language page to a new entry for "baby" in the A to Z of NHS health writing, where we could say: 

**Baby**
We use "baby" or "your baby" (not "unborn baby") in pregnancy content, from the early stages of pregnancy to birth and during the baby's first year. "Baby" is simpler than "embryo" or "foetus". [Does our user research suggest that many women already relate to it as their baby?]

For example, in our medicines content, we talk about how medicines can "affect you and your baby in pregnancy".

The words we use depend on the context, however. In content about abortion, for example, we use "pregnancy" and "ending the pregnancy". In content about miscarriage, we usually talk about "losing a pregnancy". Test with users to make sure that the words you use are right for their circumstances.

Foetus
We do not commonly use "foetus", except in the names of conditions like "foetal alcohol syndrome". We prefer "baby" [link to above].  [I had a look at our foetal alcohol syndrome page and we use "baby" there too.]

To be discussed with the wider content profession in the run up to the August Style Council meeting.

BTW Susie, I see that you spelt it fetus and I know that Jean thinks fetus is OK. My feeling is that we should leave it as foetus for now to avoid putting off the people who feel strongly that we should use the "English" spelling. Google picks it up either way. Interestingly the search on NHS.UK doesn't pick up 'fetus'. I'll see what we need to do to sort this out. ​

**Comment by sarawilcox** [comment]
Agreed preferred wording at August Style Council meeting and got clinical approval. Ready to publish on 4 September 2019.

**Comment by albfreeman** [comment]
We've had a service desk enquiry from someone suggesting the NHS website is using an outdated spelling of foetal in "[foetal alcohol spectrum disorder](https://www.nhs.uk/conditions/foetal-alcohol-spectrum-disorder/)". They've pointed us to the national [FASD clinic](https://www.fasdclinic.com/) which spells it "fetal".

I've checked the sources of our FASD content. Both [DHSC](https://www.gov.uk/government/publications/fetal-alcohol-spectrum-disorder-health-needs-assessment/fetal-alcohol-spectrum-disorder-health-needs-assessment) and [NICE](https://www.nice.org.uk/guidance/qs204) spell it "fetal". So we should probably consider updating our style to match.

**Comment by sarawilcox** [comment]
At July 2024 Style Council meeting, agreed to update our service manual entry to spell it "fetus" and "fetal": 

> **fetus** 
> 
> We spell fetus and fetal without an "o".  
> 
> We do not commonly use "fetus", except in the names of conditions like "fetal alcohol syndrome". We prefer "baby". 

Now with our clinical team for approval before we publish.

**Comment by sarawilcox** [comment]
To be published on 19 September. Approved by clinical team.
